README.md

```diff
@@ -2,5 +2,16 @@ gl-heatmap2d
 ============
 2D heatmaps
 
+Modified by: Louise Ord 
```

Comment:
How about mentioning this info in a sentence right after the following line?
`This is achieved by introducing an option, zsmooth, that defaults to the smoothed heatmap.`